### PR TITLE
added endlessrequiem.md

### PR DESCRIPTION
**Pull request by endlessrequiem**
https://raw.githack.com/endlessrequiem/endlessrequiem.github.io/aw-branch/#!pages/vi/profiles/endlessrequiem.md

### CODE REVIEW

**Comment by khalkmq** [comment]
This link does not work. Take a look at the link and replace "YourUserName" with "endlessrequiem" and replace "YourBranchName" with the name of the branch you created in your forked repository.

**Comment by endlessrequiem** [comment]
@khalkmq fixed it

**Comment by perryc85** [comment]
@endlessrequiem - Good start so far. Please use Markdown for your name versus HTML. An example would be something like `*your name*` instead of `<em>Austin White</em>`.

File: pages/vi/profiles/endlessrequiem.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Austin White
+## California, Pacific Standard Time
+### MacOS Catalina
+
+
+My name is <em>Austin White</em>, but you can call me Austin, I am a **Google Developer Group Organizer** as well as a **Google Developer Student Club Lead**. I am also the **current Director of StanHacks**, Stan State’s annual Hackathon. Aside from organizing, I am currently a **Student Assistant for Stanislaus State SERCSA**, and I am doing research on how students interact with Chatbots while enrolled in online classes.
```
Comment (lmmrssa) [inline pages/vi/profiles/endlessrequiem.md]: @endlessrequiem all good, try to use markdown tag for `<em>`

**Comment by singharpita** [comment]
Old PR closing

**Pull request closed without merging**